chromadb/test/property/test_embeddings.py

```diff
@@ -366,21 +379,24 @@ def wait_for_compaction(self) -> None:
 
     @rule(
         target=embedding_ids,
-        record_set=strategies.recordsets(collection_st),
+        record_set=strategies.recordsets(collection_st, can_ids_be_empty=True),
     )
     def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID]:
         res = super().add_embeddings(record_set)
         normalized_record_set: strategies.NormalizedRecordSet = invariants.wrap_all(
             record_set
         )
+
+        ids = normalized_record_set["ids"] or []
```

Comment:
By now we have the ids from `add_embeddings` even if the ids on `record_set` are generated `None` - we have them in `res`